core/controllers/topic_viewer.py

```diff
@@ -73,9 +73,15 @@ def get(self, topic_name):
             summary.to_human_readable_dict() for summary
             in additional_story_summaries]
 
+        skill_ids = topic.get_all_skill_ids()
+
+        subtopics = topic.get_all_subtopics()
+
         self.values.update({
```

Comment:
Can you add a test to make sure all skills and subtopics related to the topic are actually returned?

Reply:
Done!